ui/js/treeherder.js

```diff
@@ -1,6 +1,3 @@
 import angular from 'angular';
-import ngSanitize from 'angular-sanitize';
 
-export default angular.module('treeherder', [
-  ngSanitize,
```

Comment:
This file is used by log viewer/perfherder too.

Opening the log viewer for a job, I get this in the console:
`Error: "[$sce:unsafe] Attempting to use an unsafe value in a safe context.`

I believe this is because the `ng-bind-html` here relies on `$santitize`:
https://github.com/mozilla/treeherder/blob/264a4ad4fa532a92cb1adb99f97c9255593e87a5/ui/logviewer.html#L120

Let's:
* check that perfherder/the jobs view doesn't have similar exceptions
* presuming not, then we can move the `ngSanitize` dep to just the `ui/js/logviewer.js` entry point, to save all the others from having to load it (and then in the future, remove it there too)

Reply:
..er...  I mean they have HAD ``line.url``s...